src/App.tsx

```diff
@@ -1,61 +1,127 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const renderedTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState(renderedTodos);
+  const [title, setTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState(0);
+  const [titleError, setTitleError] = useState(false);
```

Comment:
Naming conventions are important in programming because they make code more readable and easier to understand. Here are some common naming conventions for boolean variables and functions:

**Boolean Variables:**
- Start with “is”, “has”, “should”, or “does” to indicate that the variable represents a true or false condition.
- Use descriptive names that accurately convey the meaning of the variable.
- Use `camelCase` for multi-word names (e.g. “isCompleted” instead of “is_completed”).

Examples:
```
isLoggedin
hasPermission
shouldShowDialog
doesExist
```

**Functions:**
- Start with a verb that describes the action the function performs.
- Use descriptive names that accurately convey the purpose of the function.
- Use `camelCase` for multi-word names.

Examples:
```
handleClick
validateForm
renderListItems
deleteAccount
```

These naming conventions help to make code more readable and self-explanatory, which is especially important when working on larger projects with other developers.

#naming